tests/CommunityToolkit.Mvvm.UnitTests/Test_AsyncRelayCommand{T}.cs

```diff
@@ -223,4 +223,95 @@ async void TestCallback(Action throwAction, Action completeAction)
 
         Assert.IsTrue(success);
     }
+
+    public void Test_AsyncRelayCommand_ExecuteDoesNotRaisesCanExecuteChanged()
+    {
+        TaskCompletionSource<object?> tcs = new();
+
+        AsyncRelayCommand<string> command = new(s => tcs.Task, allowConcurrentExecutions: true);
+
+        (object? Sender, EventArgs? Args) args = default;
+
+        command.CanExecuteChanged += (s, e) => args = (s, e);
+
+        Assert.IsTrue(command.CanExecute(""));
+
+        command.Execute("");
+
+        Assert.IsNull(args.Sender);
+        Assert.IsNull(args.Args);
+
+        Assert.IsTrue(command.CanExecute(""));
+
+        tcs.SetResult(null);
+    }
+
+    [TestMethod]
+    public void Test_AsyncRelayCommand_ExecuteWithoutConcurrencyRaisesCanExecuteChanged()
+    {
+        TaskCompletionSource<object?> tcs = new();
+
+        AsyncRelayCommand<string> command = new(s => tcs.Task, allowConcurrentExecutions: false);
+
+        (object? Sender, EventArgs? Args) args = default;
+
+        command.CanExecuteChanged += (s, e) => args = (s, e);
+
+        Assert.IsTrue(command.CanExecute(""));
+
+        command.Execute("");
+
+        Assert.AreSame(command, args.Sender);
+        Assert.AreSame(EventArgs.Empty, args.Args);
+
+        Assert.IsFalse(command.CanExecute(""));
+
+        tcs.SetResult(null);
+    }
+
+    [TestMethod]
+    public void Test_AsyncRelayCommand_ExecuteDoesNotRaisesCanExecuteChanged_WithCancellation()
```

Comment:
Minor typo, "Raises" -> "Raise"

Reply:
Whoops! Fixed in https://github.com/CommunityToolkit/dotnet/pull/111/commits/105e31ac8713124bd55351cf586c88bb9627eb18, thanks!